### DISCUSSION TITLE
Search filters (EXE, APK, ... etc)

### DISCUSSION THREAD

**Discussion by UserAccount123** [community/community#138180] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Search filters (EXE, APK, .. etc): Please add this search filter in github: showing repositories those in their (Releases/Tags) page has: (EXE, MSI, BAT, .. etc), (APK, APKS, XAPK, JAR); in case if someone looking for software/App to download .. this filter saves time by avoiding opening repositories with no (EXE, APK, .. etc) [save time = the user as usual opens a repository, scrolling down to (Releases/Tags), opening it, start to find if there is EXE, APK, .. etc, If he didn't find them then he closes the tab or go back, then go back again to search results tab. BUT this filter is a shortcut way].

**Comment by thomassiard** [comment]
Submit Feedback to GitHub: You can share this suggestion directly with GitHub via their [official feedback form](https://support.github.com/contact/feedback) under the “Product Feedback” category. Describe your use case in detail, as you did here, to illustrate how this feature would benefit users.

GitHub Community Discussions: Post your idea in [GitHub Community Discussions](https://github.com/orgs/community/discussions). Other users who find this feature useful can upvote your post or provide additional input, which could catch the attention of GitHub developers.

Use Advanced Search and Filters Temporarily: While GitHub doesn’t currently support filtering by release asset types, some third-party sites and GitHub search operators can help you narrow down results. For example:

Use keywords like "filename:exe" or "filename:apk" in the search bar alongside the name of the app you’re searching for.
Explore other file indexing tools that may search GitHub releases (although these may not be as precise or official).
